internal/producer.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"github.com/ONSdigital/dp-observation-importer/event"
+	"github.com/ONSdigital/dp-observation-importer/schema"
+	"github.com/ONSdigital/go-ns/kafka"
+	"time"
+)
+
+func main() {
+	var brokers []string
+	brokers = append(brokers, "localhost:9092")
+
+	producer := kafka.NewProducer(brokers, "observation-extracted", int(2000000))
+
+	event1 := event.ObservationExtracted{InstanceID: "7", Row: "5,,sex,male,age,30"}
+	sendEvent(producer, event1)
+	//time.Sleep(time.Duration(5000 * time.Millisecond))
```

Comment:
Removed commented code - I also moved this to the CMD directory as it is a standalone app.